src/js/views/lutpicker.js

```diff
@@ -95,24 +125,8 @@ var LutPickerView = Backbone.View.extend({
             this.success = options.success;
         }
 
-        this.loadLuts().then(data => {
-            this.luts = data.luts;
-            this.lut_names = data.png_luts;
-            this.is_dynamic_lut = Boolean(data.png_luts_new);
-            if (this.is_dynamic_lut){
-                this.luts = this.luts.map(function(lut) {
-                    lut.png_index = lut.png_index_new;
-                    return lut;
-                });
-                this.lut_names = data.png_luts_new;
-                this.lutsPngUrl = `${WEBGATEWAYINDEX}luts_png/`;
-            } else {
-                this.lutsPngUrl = STATIC_DIR + lutsPng;
-            }
-            this.lut_names = this.lut_names.map(lut_name => lut_name.split('/').pop());
-
-            this.render();
-        });
+        this.loadLuts();
+        this.render();
```

Comment:
This change now calls `this.render()` before the luts have loaded. Haven't tested to see if this actually causes a bug, but I think this will be fixed with:
```
this.loadLuts().then(() => this.render());
```

Reply:
That may indeed, I added your fix. Thanks